packages/react/test/specs/lib/felaInvokeKeyframesPlugin-test.ts

```diff
@@ -56,4 +56,12 @@ describe('felaRenderKeyframesPlugin', () => {
       animationDuration: '2s',
     })
   })
+
+  test('does not transform a list of strings', () => {
```

Comment:
Maybe the `felaInvokeKeyframesPlugin` fix and test should be moved into a separate commit? The reason for this fix is that screener tests are failing, if the order of plugins is also changed.

Reply:
The `checkboxStyles` definition contains `root: display: ['inline-grid', '-ms-inline-grid']`. This gets processed by felaInvokeKeyframesPlugin incorrectly, rendering it as `root: display: { 0: 'inline-grid', 1: '-ms-inline-grid' }.